sdk/healthdataaiservices/Azure.Health.Deidentification/samples/README.md

```diff
@@ -0,0 +1,17 @@
+---
+page_type: sample
+languages:
+- csharp
+products:
+- azure
+- azure-health-deidentification
```

Comment:
I've emailed the Learn Information Architecture team about adding this value to the allow list

Reply:
Pivoted to using HDS slug. Resolving

azure-health-data-services